tests/engines/registries/acceptance/overview/topbar-test.ts

```diff
@@ -74,6 +74,17 @@ module('Registries | Acceptance | overview.topbar', hooks => {
         assert.dom('[data-test-topbar-share-bookmark-fork]').isVisible();
         assert.dom('[data-test-topbar-states]').isVisible();
 
+        const anonymousReg = server.create('registration', {
+            isAnonymous: true,
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+        }, 'isWithdrawn');
```

Comment:
```suggestion
        });
```
probably more important to check non-withdrawn anonymous

Reply:
oh wait there's an [`anonymized` trait](https://github.com/CenterForOpenScience/ember-osf-web/blob/39ded07ee9ee2aa3aa3cd379d960672867041c78/mirage/factories/node.ts#L176-L178) -- should use that instead of `isAnonymous` (and might address that test failure)

Reply:
Test failure was me not changing after copy-pasting 😅 
I tried the `anonymized` trait, but it didn't seems to change `registration.isAnonymous` properly. It doesn't seem to be used anywhere, so I could update it so it adds `anonymous: true` to the apiMeta?

Reply:
ohh i was looking at the [`node` serializer, which does set `meta.anonymous` based on the `_anonymous` attr from](https://github.com/CenterForOpenScience/ember-osf-web/blob/39ded07ee9ee2aa3aa3cd379d960672867041c78/mirage/serializers/node.ts#L187) -- but i guess the mirage serializer is one place registrations aren't nodes 🤷 

so yeah i think best would be updating the mirage registration serializer with a similar `buildApiMeta` (or even move the anonymized check to the [application serializer](https://github.com/CenterForOpenScience/ember-osf-web/blob/39ded07ee9ee2aa3aa3cd379d960672867041c78/mirage/serializers/application.ts#L46-L50)? unsure)

Reply:
I wasn't having any luck with creating a registration with the `anonymized` trait and adding `buildApiMeta` to the registration serializer (`apiMeta: undefined`, `isAnonymous: false` in ember inspector), so I've tested out a node with the `anonymized` trait. It seems like using that trait doesn't anonymize the node either. I'm not too sure where the exact problem is, and it feels like it's something simple I'm overlooking... any ideas?